phpstan.neon.dist

```diff
@@ -1,4 +1,13 @@
+includes:
+    - vendor/szepeviktor/phpstan-wordpress/extension.neon
 parameters:
-	level: 5
-	paths:
-		- src
+#    level: 5
+    level: 0
```

Comment:
Hi Viktor, I want to have level 5 executed, but the configuration has been changed to 0. Is your idea to start with 0, fix all the errors, then switch to 1, fix all the errors, and so on until reaching 5?

Reply:
Yes!